CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,54 @@
+# **CODE OF CONDUCT**
```

Comment:
@EmmanuelTheCoder you're on a roll 💪. Thank you.

This file shouldn't exist in this PR. It appears you created this branch from the `code-of-conduct` branch instead of `master`.

Please check out a new branch from `master`, add the summary there, and create a new PR.

I'm going to close this PR now.

Reply:
I have just done that. But unfortunately I dont know how to use the session-summary_pull-request_template when creating a PR